base/src/main/java/org/kontinuity/catapult/base/identity/IdentityVisitor.java

```diff
@@ -0,0 +1,15 @@
+package org.kontinuity.catapult.base.identity;
+
+/**
+ * @author <a href="mailto:ggastald@redhat.com">George Gastaldi</a>
+ */
+public interface IdentityVisitor {
+
+    default void visit(TokenIdentity token) {
```

Comment:
Sure, and my point is that unsupported things are better not exposed by the API.